src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/RefreshConstants.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using System;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    internal class RefreshConstants
+    {
+        // Key-values
+        public static readonly TimeSpan DefaultCacheExpirationInterval = TimeSpan.FromSeconds(30);
+        public static readonly TimeSpan MinimumCacheExpirationInterval = TimeSpan.FromMilliseconds(1000);
+
+        // Feature flags
+        public static readonly TimeSpan DefaultFeatureFlagsCacheExpirationInterval = TimeSpan.FromSeconds(30);
+        public static readonly TimeSpan MinimumFeatureFlagsCacheExpirationInterval = TimeSpan.FromMilliseconds(1000);
+
+        // Key Vault secrets
```

Comment:
TimeSpan.FromMilliseconds(1000) can use timespan.fromseconds(1). Then it's all seconds, no ms in the picture.

Reply:
updated